types/trtc-js-sdk/index.d.ts

```diff
@@ -0,0 +1,756 @@
+// Type definitions for trtc-js-sdk 4.3
+// Project: https://github.com/tencentyun/TRTCSDK#readme
+// Definitions by: yokots <https://github.com/yokots>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+// tslint:disable-next-line:export-just-namespace
```

Comment:
Could you please enable the tslint rule back. There's no need to export only a namespace. Instead use export on the objects you want to export.

Reply:
this is an umd module

Reply:
I also enable esModuleInterop in tsconfig.json

Reply:
The `export as namespace` already does the job of exporting for umd. Grouping all of the interfaces inside the a `namespace` is unnecessary. You can read more about the rule on the tslint site.

Reply:
when remove `export = xxx`,  the typescript report some errors, `Global module exports may only appear in module files.` and `Duplicate identifier Omit`, I write this definition just like `react` and other modules, I don't known how to fix it. And apologize for my poor english.

Reply:
Sorry for not being descriptive enough. You'll need to remove `export =` and `declare namespace TRTC`, as you are already exporting the namespace by using `export as namespace TRTC`. Then, prepend `export` on every object the library has exports like `export namespace Logger`, `export function checkSystemRequirements(): Promise<boolean>;`, etc.

Hope this makes sense.

Reply:
Thanks for Detailed explanation，I have fixed it.